src/azurelinux/3.0/net10.0/crossdeps-builder/amd64/Dockerfile

```diff
@@ -56,11 +56,11 @@ RUN wget https://releases.llvm.org/release-keys.asc && \
     gpg --import release-keys.asc && \
     rm release-keys.asc && \
 # 2. Download llvm sources and signature, and verify signature
-    LLVM_VERSION=19.1.6 && \
+    LLVM_VERSION=20.1.0-rc2 && \
```

Comment:
Why not use https://github.com/llvm/llvm-project/releases/tag/llvmorg-20.1.0 now that it's already released?

Reply:
Good catch @am11!